base/replutil.jl

```diff
@@ -328,9 +328,19 @@ function show_method_candidates(io::IO, ex::MethodError)
     end
 end
 
-function show_trace_entry(io, fname, file, line, n)
+function show_trace_entry(io, fname, file, line, inlinedfile, inlinedline, n)
     print(io, "\n")
-    print(io, " in ", fname, " at ", file)
+    print(io, " in ", fname)
+
+    if (inlinedfile != symbol(""))
+        print(io, " at ", inlinedfile, ":", inlinedline)
+        print(io, " \n    [ thrown by inlined function defined at ]\n    ")
```

Comment:
I find this output a bit confusing. Since the brackets close before all the info is printed, it looks like the location is simply missing. I would print it something like `in inlined function defined at file:line`.